app/components/ErrorBoundary.tsx

```diff
@@ -1,5 +1,5 @@
 import { isRouteErrorResponse, useRouteError, Link, type ErrorResponse } from "react-router";
-import { Card, CardContent, CardHeader, CardTitle } from "~/components/ui/card";
+import { Card } from "react-daisyui";
```

Comment:
is this working for you? 

`react-daisyui doesn't support the daisyui5. It needs daisyui 4.`

Reply:
yes, we are using direct daisyu ui classess now.

https://daisyui.com/components/card/